.local.dic

```diff
@@ -103,3 +103,5 @@ VMware
 WebStorm
 Webstorm's
 WSL
+monorepo
+workspaces
```

Comment:
Can you add these words so that the list is kept in alphabetical order (case-insensitive)?